ui/app/components/ui/popover/popover.component.js

```diff
@@ -1,13 +1,25 @@
 import React, { PureComponent } from 'react'
 import ReactDOM from 'react-dom'
 import PropTypes from 'prop-types'
-import PopoverHeader from './popover.header.component'
 
-const Popover = ({ title, children, onClose }) => (
+const Popover = ({ title, subtitle, children, onBack, onClose }) => (
   <div className="popover-container">
     <div className="popover-bg" onClick={onClose} />
     <div className="popover-wrap">
-      <PopoverHeader title={title} onClose={onClose} />
+      <header className="popover-header">
+        <div className="popover-header__title">
+          <h2 title={title}>
+            {
+              onBack
+                ? <i className="fas fa-chevron-left" onClick={onBack} />
+                : null
+            }
+            {title}
+          </h2>
+          <i className="fas fa-times" onClick={onClose} />
```

Comment:
This can no longer be focused, now that it's an `i` instead of a `button`

Reply:
The same goes for the back button as well

Reply:
Updated, and now with titles as well